### PR TITLE
Analyzer and Anonymizer Python 3.9 support

### PR DESCRIPTION
**Pull request by SharonHart**
Analyzer an Anonymizer to support python 3.9.
Image redactor will be in a different iteration as it needs Analyzer version to be bumped in pypi.
Essentially, spacy was the blocker, which was updated to the first version supporting 3.9 (we might keep upgrading in another iteration)

### CODE REVIEW

**Comment by omri374** [comment]
@SharonHart if the issue is with the demo website's text, we can consider changing it a little bit.

**Comment by SharonHart** [comment]
@omri374 Apparently [iban is a nationality](https://en.wikipedia.org/wiki/Iban_people) :)

File: e2e-tests/resources/demo.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ On September 18 I visited microsoft.com and sent an email to test@presidio.site,
 
 My passport: 191280345 and my phone number: (212) 555-1234.
 
-This is a valid IBAN: IL150120690000003111111 . Can you please check the status on bank account 954567876544?
+This is a valid International Bank Account Number: IL150120690000003111111 . Can you please check the status on bank account 954567876544?
```
Comment (SharonHart) [inline e2e-tests/resources/demo.txt:10]: Needs to be changed in the website as well

**Comment by navalev** [comment]
Let's remove the 3.6 support once this is added

File: e2e-tests/resources/demo.txt (hunk 1 above)
Comment (omri374) [reply e2e-tests/resources/demo.txt:10]: Worth doing a PR and linking it here so we won't forget.

**Review by omri374 — APPROVED** [review]

**Review by navalev — APPROVED** [review]

**Pull request merged**